src/oscar/apps/order/abstract_models.py

```diff
@@ -1200,3 +1208,29 @@ def description(self):
         if self.voucher_code:
             return self.voucher_code
         return self.offer_name or ""
+
+
+class AbstractSurcharge(models.Model):
+    order = models.ForeignKey(
+        'order.Order',
+        on_delete=models.CASCADE,
+        related_name="surcharges",
+        verbose_name=_("Surcharges"))
+
+    name = models.CharField(
+        _("Surcharge"), max_length=128
+    )
+
+    description = models.TextField(_('Description'), blank=True)
```

Comment:
Is this field used anywhere?

Reply:
No it is not. I removed it.

Reply:
I also removed it from the the Base object in apps.checkout.surcharges